src/app/products/products.component.html

```diff
@@ -2,45 +2,39 @@
 <div id="entries" class="container selectors">
   <label class="lable">Show</label>
   <select class="form-control">
-    <option value="10">10</option>
-    <option value="25">25</option>
-    <option value="50">50</option>
-    <option value="100">100</option>
+      <option *ngFor="let entry of entries" [value]=entry.id>{{entry.value}}</option>
+
   </select>
   <label class="lable">entries</label>
   <!-- End of Entries -->
 
 
-  <!-- Fillter -->
-  <div class="fillter">
+  <!-- Filter -->
+  <div class="filter">
     <label class="lable">Supplier</label>
     <select class="form-control supplier">
-      <option value="0">All</option>
-      <option value="1">supplier1</option>
-      <option value="2">supplier2</option>
-      <option value="3">supplier3</option>
+      <option *ngFor="let supplier of suppliers" [value]=supplier.id>{{supplier.name}}</option>
```

Comment:
@BaidaaAlbzoor  please add quotations like [value]=supplier.id ==> [value]="supplier.id". 
and to type.id too